backend/src/middleware/notifications/notificationsMiddleware.spec.ts

```diff
@@ -736,6 +797,72 @@ describe('notifications', () => {
             expect(pubsubSpy).toHaveBeenCalledTimes(1)
           })
         })
+
+        describe('but the author of the post muted me', () => {
```

Comment:
```suggestion
        describe('but the author of the comment muted me', () => {
```

Reply:
No, this is the author of the post:
```
        AND NOT (user)-[:BLOCKED]-(commenter)
        AND NOT (user)-[:BLOCKED]-(postAuthor)
        AND NOT (user)-[:MUTED]->(commenter)
        AND NOT (user)-[:MUTED]->(postAuthor)
```

Reply:
The test shows, that is has no effect to the notification